docs/source/guide/dataset_manage.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Manage Datasets
+short: Manage Datasets
+tier: enterprise
+order: 0
+order_enterprise: 408
+meta_title: Manage a Dataset in Label Studio Enterprise
+meta_description: How to manage your datasets in Label Studio Enterprise 
+date: 2023-08-23 12:07:13
+hide_sidebar: true
+---
+
+
+## Dataset settings
+
+From the Datasets page, click the overflow menu next to dataset and select **Settings**.  
+
+![Overflow menu next to a dataset](/guide/images/data_discovery/dataset_settings.png)
+
+From here you can do the following:
+
+- Edit the dataset name and description.
```

Comment:
Mind linking to the section where all that gets set up? I figure when users are reading this, it's likely they've forgotten how they set everything up